### PR TITLE
Direction slice

### PR DESCRIPTION
**Pull request by darya42185**
## GitHub Board

[Direction: create state slice #88](https://github.com/ita-social-projects/dokazovi-fe/issues/88 )

## Code reviewers
- [ ] @github_username

## Summary of issue
1. - [x] Set up initial state using existing API types
2. - [x] Use Redux Toolkit to generate action creators & reducers

### CODE REVIEW

File: src/modules/direction/store/directionSlice.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  reducers: {
+    loadCourses: (state, action: PayloadAction<ICourse[]>) => {
+      state.courses = action.payload;
+    },
+  },
+});
+
+export const { loadCourses } = directionSlice.actions;
+
+export default directionSlice.reducer;
```
Comment (calster00) [inline src/modules/direction/store/directionSlice.tsx:27]: this has to be a part of `rootReducer`

**Review by SergeOlabin — CHANGES_REQUESTED** [review]
pls merge develop to local branch

File: src/lib/components/Header/ListItems.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
```
Comment (SergeOlabin) [inline src/lib/components/Header/ListItems.tsx:1]: this file was removed before, wasn't it?

File: src/store/rootReducer.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
+import directionReducer from '../modules/direction/store/directionSlice';
 import mainReducer from '../modules/main/store/mainSlice';
 
 const rootReducer = combineReducers({
   main: mainReducer,
+  direction: directionReducer,
```
Comment (SergeOlabin) [inline src/store/rootReducer.ts:7]: we need following structure for direction: `{ [%DirectionsEnumKey%]: directionReducer }`, so the overall structure will be: 
```
{
  main: MainReducer,
  directions: {
    covid19: directionReducer,
    therapy: directionReducer,
    // ...
  }
```
}

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/lib/components/Header/ListItems.tsx (hunk 2 above)
Comment (darya42185) [reply src/lib/components/Header/ListItems.tsx:1]: if you delete these commits, then this will lead to conflicts, so I think we should not delete them

**Pull request closed without merging**